__tests__/app.test.js

```diff
@@ -326,4 +327,62 @@ describe("Northcoders News API ", () => {
             });
         });
     });
-    
\ No newline at end of file
+
+
+describe("GET /api/articles with QUERIES", () => {
+  test("Should return status 200 for a valid query", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200);
+  })
+  test("Should return items only matching the topic query", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .then(({ body: { articles } }) => {
+        articles.forEach((singleArticle) => {
+          expect(singleArticle.topic).toBe("mitch")
+        })
+      })
+  });
+  test("Should return items in order specified defaulting to descending", () => {
```

Comment:
Your test description suggests a descending order to be checked, but you are testing ascending.